app/models/account-group.js

```diff
@@ -1,9 +1,8 @@
 import Model, { attr, hasMany } from '@ember-data/model';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  uri: attr(),
-  name: attr(),
-  users: hasMany('user'),
-});
+export default class AccountGroup extends Model {
+  @attr() uri;
```

Comment:
minor. When using the decorator notation without specifying a transformer, you can drop the parens.